util/topgen/templates/toplevel.h.tpl

```diff
@@ -193,6 +193,14 @@ ${helper.clkmgr_gateable_clocks.render()}
  */
 ${helper.clkmgr_hintable_clocks.render()}
 
+/**
+ * MMIO Region
+ *
+ * MMIO region excludes ROM, RAM, and flash but includes retention RAM.
```

Comment:
going by my comment above, maybe we could also re-word this to say... excludes any memory that is that is separate from the module configuration space.  So this would not include things like spi device memory, usbdev memory, so on and so forth.

Reply:
Sure, what I tried to mean was _main_ SRAM but it wasn't clear. Done.